var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -743,6 +754,16 @@ def define_enable(suffix, value=None):
                 ]
             )
 
+        if "+pytrilinos2" in spec:
+            options.append(define_trilinos_enable("PyTrilinos2", True))
+            options.append(define_from_variant("PyTrilinos2_ENABLE_TESTS", "test"))
+            binder = spec["binder"].prefix.bin.binder
+            clang_include_dirs = spec["binder"].clang_include_dirs
+            LibClang_include_dir = spec["binder"].LibClang_include_dir
```

Comment:
Did you forget to push a change to binder? also please use PEP-8 compliant variable names (`libclang_include_dir`)

Reply:
Most of the code in binder was already there but I did make the variables adhere to PEP8 in both binder/package.py and trilinos/package.py.